### ISSUE TITLE
Layout becomes messed up after reporting an issue

### ISSUE THREAD

**Issue by softvision-oana-arbuzov** [webcompat/webcompat.com#2155]
URL: https://staging.webcompat.com/
Browser / Version: Firefox Nightly Mobile 60.0a1 (2018-02-18), Firefox Nightly Desktop 60.0a1 (2018-02-16), Chrome Desktop 64.0.3282.167, Chrome Mobile 64.0.3282.137 
Operating System: Windows 10 Pro, Google Pixel (Android 8.0.0) - 1080 x 1920 pixels (~441 ppi pixel density)

**Steps to Reproduce:**
1. Navigate to https://staging.webcompat.com/
2. Tap "Report bug" button.
3. Type in all required information.
4. Tap "Report" button (anonymous of with the Github account).
5. Observe behavior after issue is logged successfully.

**Expected Behavior:**
An overlay with "Thanks" message is displayed and dismissed for after a few seconds.
"Share on Twitter"/"Share on Facebook" button are displayed under the reported issue.

**Actual Behavior:**
"Thanks for reporting an issue!" message overlaps the bug report.

**Note**
1. Workaround: Refreshing the screen, the "Thanks" message is dismissed.
2. "Thanks" message: 
```html
<div class="notification-bar js-flashmessage is-active notification-thanks">
	<h4>Thanks for reporting an issue!</h4>
	<p>You're helping us make the web a better place to work and play.</p>
	<p>Tell your friends about the bug you just filed:</p>
	<a class="wc-Button wc-Button--action" href="https://twitter.com/intent/tweet?text=I%20just%20filed%20a%20bug%20on%20the%20internet%3A&amp;url=https%3A%2F%2Fwebcompat.com%2Fissues%2F1376&amp;via=webcompat" target="_blank">Share on Twitter</a>
	<a class="wc-Button wc-Button--action" href="https://facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwebcompat.com%2Fissues%2F1376" target="_blank">Share on Facebook</a>
</div>
```

Watchers:
@softvision-sergiulogigan
@softvision-oana-arbuzov

![screenshot-1519044666800](https://user-images.githubusercontent.com/12184325/36379219-6c156598-1586-11e8-8bdb-96470fc28061.jpg)

![thanksmessageoverbugreport](https://user-images.githubusercontent.com/12184325/36379275-a31d7fd0-1586-11e8-8e82-4f14a47eba50.gif)

**Comment by zoepage** [comment]
`thanks.jst` is not refactored yet. This is what is causing this issue.

**Comment by zoepage** [comment]
merged #2278
